ruby/vscode/minitest.rb

```diff
@@ -32,6 +32,7 @@ def run(*args)
       reporter = Reporter.new
       reporter.start
       runner = Runner.new(reporter: reporter)
+      puts args
```

Comment:
Is this intended to be here? Or is this a stray from development/debugging?

Reply:
@thomthom Thanks for your review!

Actually I also think that puts was only for debugging, that's why removed it. I also revert commit 11f7896 because I didn't find any reason to add everyone ignored files to the official .gitignore in the repo.

Keep in mind that I'm not an author of all commits in this PR, so I will be grateful for any help and review :).

Reply:
I only had a quick glance. I am not familiar with how a VSCode extension work. But I was curious to how it could fix #44. Makes sense to me now that I see it's calling shell commands with environment variable syntax that doesn't work on Windows.